ensime_shared/client.py

```diff
@@ -170,6 +170,7 @@ def lazy_initialize_ensime():
 
                 try:
                     self.ensime = self.launcher.launch()
+                    self.editor.start_timer()
```

Comment:
I can't see this method implemented in `Editor` I think you just kept the old name `cursorhold`. Also I think that this method could use a more descriptive name, something like `start_refresh_timer`